.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"singleQuote": false,
```

Comment:
Can we set this to true? This is highly subjective so I'll need @saadpasta view on this too.

Reply:
I love double quotes, I mostly use it, but I'm open to suggestions.

Reply:
This is okay too I guess, no strong opinions really. Let's hold on this until #318 is in (sorry for another rebase 😅)

Reply:
@kartikcho I think #318 is closed. Can you merge this?

Reply:
I literally just merged a PR haha sorry

Reply:
will rebase it after few hours.

Reply:
I rebased it, possibly do a final check and then merge it.